### PR TITLE
Add style checks

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

**Comment by sungam3r** [comment]
~60-70 warnings fixed

File: src/GraphQL.DataLoader/DataLoaderDocumentListener.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ public Task BeforeExecutionAsync(IExecutionContext context)
         }
 
         /// <inheritdoc/>
-        public Task BeforeExecutionAwaitedAsync(IExecutionContext context)
+        public Task BeforeExecutionAwaitedAsync(IExecutionContext _)
```
Comment (Shane32) [inline src/GraphQL.DataLoader/DataLoaderDocumentListener.cs:36]: Public api should not have argument names called `_`

File: src/GraphQL.ApiTests/GraphQL.DataLoader.approved.txt [hunk 2]
```diff
@@ -58,8 +58,8 @@ namespace GraphQL.DataLoader
         public System.Threading.Tasks.Task AfterExecutionAsync(GraphQL.Execution.IExecutionContext context) { }
         public System.Threading.Tasks.Task AfterValidationAsync(GraphQL.Execution.IExecutionContext context, GraphQL.Validation.IValidationResult validationResult) { }
         public System.Threading.Tasks.Task BeforeExecutionAsync(GraphQL.Execution.IExecutionContext context) { }
-        public System.Threading.Tasks.Task BeforeExecutionAwaitedAsync(GraphQL.Execution.IExecutionContext context) { }
-        public System.Threading.Tasks.Task BeforeExecutionStepAwaitedAsync(GraphQL.Execution.IExecutionContext context) { }
+        public System.Threading.Tasks.Task BeforeExecutionAwaitedAsync(GraphQL.Execution.IExecutionContext _) { }
+        public System.Threading.Tasks.Task BeforeExecutionStepAwaitedAsync(GraphQL.Execution.IExecutionContext _) { }
```
Comment (Shane32) [inline src/GraphQL.ApiTests/GraphQL.DataLoader.approved.txt:62]: Public api should not have arguments named `_`

File: src/GraphQL/Utilities/SchemaBuilder.cs [hunk 3]
```diff
@@ -199,8 +199,8 @@ protected virtual void PreConfigure(Schema schema)
 
         protected virtual IGraphType GetType(string name)
         {
-            _types.TryGetValue(name, out IGraphType type);
-            return type;
+            _types.TryGetValue(name, out var type);
+            return type!;
```
Comment (Shane32) [inline src/GraphQL/Utilities/SchemaBuilder.cs:203]: Seems like either it should be nullable return value or else use the Item property so an exception is thrown if the key doesn’t exist. Probably do the former and then change to the latter in a separate PR

**Review by Shane32 — CHANGES_REQUESTED** [review]

File: src/GraphQL.ApiTests/GraphQL.DataLoader.approved.txt (hunk 2 above)
Comment (Shane32) [reply src/GraphQL.ApiTests/GraphQL.DataLoader.approved.txt:62]: We should check why this method BeforeExecutionStepAwaitedAsync exists in the develop branch - I thought it was removed when the obsolete members was removed. Maybe I missed it.

**Comment by sungam3r** [comment]
Done

**Review by Shane32 — APPROVED** [review]

**Pull request merged**